README.md

```diff
@@ -202,14 +215,14 @@ vars:
 ``` 
 </details>
 
-## (Optional) Step 6: Orchestrate your models with Fivetran Transformations for dbt Core™
+## (Optional) Step 7: Orchestrate your models with Fivetran Transformations for dbt Core™
 <details><summary>Expand for details</summary>
 <br>
 
 Fivetran offers the ability for you to orchestrate your dbt project through [Fivetran Transformations for dbt Core™](https://fivetran.com/docs/transformations/dbt). Learn how to set up your project for orchestration through Fivetran in our [Transformations for dbt Core setup guides](https://fivetran.com/docs/transformations/dbt#setupguide).
 </details>
 
-## (Optional) Step 7: Validate your data
+## (Optional) Step 8: Validate your data
```

Comment:
I don't believe this is needed anymore and can be `7` as we are no longer making a separate step for these variables.

Reply:
Fixed.